elixir/cowboy/mix.exs

```diff
@@ -3,33 +3,14 @@ defmodule Server.MixProject do
 
   def project do
     [
-      # App config
       app: :server,
       version: "0.1.0",
-
-      # Elixir config
-      elixir: "~> 1.10",
+      elixir: "~> 1.14",
```

Comment:
```suggestion
      elixir: "~> 1.15",
```

Reply:
Elixir 1.15 is still in release candidate state so it wouldn't work. Once it is released it can be updated, but it is not hugely important since this just indicates MINIMUM Elixir version required to run the project.

Reply:
then we should probably not use 1.15 in here
https://github.com/Sgiath/web-frameworks/blob/1b77fa8b00039bbc00351b996502448bc744ee53/elixir/Dockerfile#L1

Reply:
It was updated here https://github.com/the-benchmarker/web-frameworks/pull/6389

The 1.15 official release is matter of days now so I wouldn't stress too much about it.

Reply:
ok, maybe the above PR was fired too early then. I do not know how elixir is officially released :-)